src/fprime/cookiecutter_templates/cookiecutter-fprime-deployment/{{cookiecutter.deployment_name}}/Top/{{cookiecutter.deployment_name}}Topology.cpp

```diff
@@ -143,6 +143,8 @@ void setupTopology(const TopologyState& state) {
     setBaseIds();
     // Autocoded connection wiring. Function provided by autocoder.
     connectComponents();
+    // Autocoded configuration. Function provided by autocoder.
+    configComponents(state);
```

Comment:
@LeStarch this has been fixed in https://github.com/nasa/fprime-tools/pull/206/files already

Should the call be made before `configureTopology()` ? In which case it should be removed line 153 (further below)

Reply:
Yes, I believe so.  The phase code cannot, by definition, know or care what the hand config does.  As such, I assumed it was safest to place this first.

Reply:
@thomas-bc fixed!  What was the reason to place this after register commands?